unicodetools/src/test/java/org/unicode/test/TestSecurity.java

```diff
@@ -524,9 +526,9 @@ public void TestWholeScripts() {
             {"⼒", Status.SAME}, // KANGXI RADICAL POWER
             {"力", Status.SAME}, // CJK UNIFIED IDEOGRAPH-529B
             {"!", Status.SAME, Status.OTHER},
-            {"\u0300", Status.SAME},
+            {"\u0300", Status.COMMON},
             {"a\u0300", Status.SAME, Status.COMMON, Status.OTHER},
-            {"ä", Status.SAME, Status.COMMON, Status.OTHER},
+            {"ä", Status.SAME, Status.OTHER},
```

Comment:
I haven't looked at what these expected values are, but (a) I assume these do naturally fall out from the data changes and (b) I believe that whole-script confusables haven been put on ice until someone can figure out real use cases, so this probably does not matter a lot.

FYI @macchiati